Content.Client/CartridgeLoader/Cartridges/LogProbeUiFragment.xaml.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Content.Shared.CartridgeLoader.Cartridges;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.CartridgeLoader.Cartridges;
+
+[GenerateTypedNameReferences]
+public sealed partial class LogProbeUiFragment : BoxContainer
+{
+    private readonly StyleBoxFlat _styleBox = new()
+    {
+        BackgroundColor = Color.Transparent,
+        BorderColor = Color.FromHex("#5a5a5a"),
+        BorderThickness = new Thickness(0, 0, 0, 1)
+    };
+
+    public LogProbeUiFragment()
+    {
+        RobustXamlLoader.Load(this);
+        Orientation = LayoutOrientation.Vertical;
+        HorizontalExpand = true;
+        VerticalExpand = true;
+        HeaderPanel.PanelOverride = _styleBox;
+    }
+
+    public void UpdateState(List<PulledAccessLog> logs)
+    {
+        ProbedDeviceContainer.RemoveAllChildren();
+
+        //Reverse the list so the oldest entries appear at the bottom
+        logs.Reverse();
+
+        //Enable scrolling if there are more entries that can fit on the screen
+        ScrollContainer.HScrollEnabled = logs.Count > 9;
+
+        foreach (var log in logs)
+        {
+            AddAccessLog(log);
+        }
+    }
+
+    private void AddAccessLog(PulledAccessLog log)
```

Comment:
I feel like this is just a hardcoded control, since the only dynamic bit are timeLabel.Text and accessorLabel.Text just make a AccessLogEntry xaml control or something